### PR TITLE
Fix usage of media functions

### LINKED ISSUE #2165: Uncaught PHP Exception Twig\Error\SyntaxError: "Unexpected "thumbnail" tag
Matched route "admin_app_media_sonatamediamedia_list".

Uncaught PHP Exception Twig\Error\SyntaxError: "Unexpected "thumbnail" tag (expecting closing tag for the "if" tag defined near line 34)." at /var/www/html/vendor/sonata-project/media-bundle/src/Resources/views/Form/media_widgets.html.twig line 35

trace:

```
/var/www/html/vendor/sonata-project/media-bundle/src/Resources/views/Form/media_widgets.html.twig:35 {▼
        Twig\Parser->subparse($test, bool $dropNeedle = false): Node …
        › <div class="pull-left" style="margin-right: 5px">
        ›     {% thumbnail value, 'admin' with {'class': 'img-polaroid media-object'} %}
        › </div>
      }
```

version:

```
"name": "sonata-project/media-bundle",
            "version": "4.x-dev",
            "source": {
                "type": "git",
                "url": "https://github.com/sonata-project/SonataMediaBundle.git",
                "reference": "7e503807f85a9dc1b6469c7b2454890e43354ed6"
```

### PR DESCRIPTION
**Pull request by jordisala1991**
## Subject

I am targeting this branch, because this is BC.

Closes #2165 

## Changelog

```markdown
### Fixed
- Fixed usage of deprecated custom twig tags.
```

### CODE REVIEW

File: src/Resources/views/MediaAdmin/list_image.html.twig [hunk 1]
```diff
@@ -12,5 +12,11 @@ file that was distributed with this source code.
 {% extends '@SonataAdmin/CRUD/base_list_field.html.twig' %}
 
 {% block field %}
-    <a href="{{ admin.generateUrl(route, {'id' : object|sonata_urlsafeid }) }}">{% thumbnail attribute(object, field_description.name ), 'admin' with {'width': 75, 'height': 60} %}</a>
+    {% set media = attribute(object, field_description.name) %}
+
+    {% if media is not null %}
+        <a href="{{ admin.generateObjectUrl(route_name, object) }}">
```
Comment (jordisala1991) [inline src/Resources/views/MediaAdmin/list_image.html.twig]: I am not sure about this one, can this be applied to Sonata 3? @VincentLanglet

Comment (VincentLanglet) [reply src/Resources/views/MediaAdmin/list_image.html.twig]: Seems ok to me. But this can be done in `inner_row_media.html.twig` too isn't it ?

Comment (jordisala1991) [reply src/Resources/views/MediaAdmin/list_image.html.twig]: Fixed.

**Review by core23 — APPROVED** [review]

**Review by VincentLanglet — APPROVED** [review]

**Comment by sarim** [comment]
Hopefully it gets merged into 4.x branch too. I'm using my own fork for now in a symfony 5 project.

**Pull request merged**